components/form/CheckboxGroup.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { useFormContext } from "react-hook-form";
+import { Label, HelperLabel } from "./Label";
+import { ErrorMessage } from "./ErrorMessage";
+import classNames from "classnames";
+import { FEEDBACK } from "@/utils/constant";
+
+type CheckboxProps = {
```

Comment:
Please don't forget to add the optional properties here too.

Reply:
Noted Fixed now